go2html.go

```diff
@@ -46,7 +48,7 @@ const (
 
 var safeTextReplacer = strings.NewReplacer("<", "&lt;", ">", "&gt;", "\"", "&quot", "'", "&quot")
 
-func NewTemplate(key string, configure func(*TemplateCfgr)) *Template {
+func NewTemplate(rc *report.RContext, key string, configure func(*TemplateCfgr)) *Template {
 	fs := []interface{}{}
 	t := &Template{
```

Comment:
- Template has report field, but here we don't have it.
- NewTemplate() can create report within itself, there is no need to pass it as an argument. TemplateCfgr should use different report.